storage/api/README.md

```diff
@@ -10,3 +16,31 @@ These samples are used on the following documentation pages:
 * https://cloud.google.com/docs/authentication
 
 <!-- end-auto-doc-link -->
+
+## Running the samples
+
+1. Your environment must be setup with [authentication
+information](https://developers.google.com/identity/protocols/application-default-credentials#howtheywork). If you're running in your local development environment and you have the [Google Cloud SDK](https://cloud.google.com/sdk/) installed, you can do this easily by running:
+
+        $ gcloud init
+
```

Comment:
is this our standard now? What I dislike about it is it only really makes sense the first time, if you already have a configuration, it's going to create a new one every time.